### ISSUE TITLE
exclude_folder_pattern not to be too slow in big folders

### ISSUE THREAD

**Issue by Cottin** [jfromaniello/sublime-node-require#15]
This plugin works very nicely in smaller folders!

...but when you have too big of a folder the plugin takes 10 seconds to generate the list of file suggestions which just makes it impossible to use.

What do you say to the idea of having `exclude_folder_pattern` in the `.sublime-settings` file like https://www.sublimetext.com/docs/2/projects.html ?

**Comment by jfromaniello** [comment]
Yes, i like it. Do you know how to read `.sublime-settings` from the plugin api?
